### PR TITLE
Replace instrumentation module/package instances with instrumentation library

### PR DESCRIPTION
**Pull request by svrnm**
This PR replaces some instances where "(auto) instrumentation module" or "(auto) instrumentation package" can be replaced with "instrumentation library"

### CODE REVIEW

**Review by tsloughter — APPROVED** [review]

File: content/en/blog/2022/dotnet-instrumentation-first-beta.md
Comment (Kielek) [inline content/en/blog/2022/dotnet-instrumentation-first-beta.md:17]:
```suggestion
ASP.NET Core requests, you need to use the ASP.NET Core instrumentation library
```

**Review by pichlermarc — APPROVED** [review]

File: content/en/docs/languages/net/libraries.md
Comment (tiffany76) [inline content/en/docs/languages/net/libraries.md:54]:
```suggestion
Then you can install the instrumentation libraries:
```

File: content/en/docs/languages/php/getting-started.md
Comment (tiffany76) [inline content/en/docs/languages/php/getting-started.md:145]:
```suggestion
With the OpenTelemetry PHP extension set up and an instrumentation library
```

**Review by tiffany76 — APPROVED** [review]

**Review by theletterf — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Comment by theletterf** [comment]
@svrnm Looks like we only need @open-telemetry/php-approvers approval.

**Pull request merged**
